paths/paths.go

```diff
@@ -62,16 +63,17 @@ func Exercises(path string) string {
 	return expandPath(path)
 }
 
-// Recalculate sets exercism paths based on Home.
-func Recalculate() {
-	if Home == "" {
-		home, err := findHome()
-		if err != nil {
-			panic(err)
-		}
+// Recalculate sets exercism paths
+func Recalculate(home, configHome string) {
```

Comment:
I agree, this function doesn't have a lot of value right now, especially given that we now have other inputs for doing the testing.  I think we could justify keeping it if we gave it a better name. Maybe `assignConfigHome` or `resolveConfigHome` (if we assign `paths.ConfigHome` outside the function). if we don't fold it in to `init`, I don't think it needs to be exported.